code/modules/hydroponics/seeds.dm

```diff
@@ -441,65 +478,28 @@
  * Sets the plant's weed growth percentage to the value of adjustamt. (Max 67%)
  */
 /obj/item/seeds/proc/set_weed_chance(adjustamt)
-	weed_chance = clamp(adjustamt, 0, 67)
+	weed_chance = clamp(adjustamt, 0, MAX_PLANT_WEEDCHANCE)
 	var/datum/plant_gene/core/C = get_gene(/datum/plant_gene/core/weed_chance)
 	if(C)
 		C.value = weed_chance
 
+/**
+ * Override for seeds with unique text for their analyzer. (No newlines at the start or end of unique text!)
+ * Returns FALSE if no unique text, or a string of text if there is.
```

Comment:
This should probably be if null is returned instead, using FALSE to mean nothing feels weird.